CMakeLists.txt

```diff
@@ -1612,6 +1612,10 @@ if (BUILD_TESTS)
   include(CTest)
 endif (BUILD_TESTS)
 
+if (AARCH64)
+  option(BUILD_TESTS_SVE "Build tests with SVE enabled and high optimization (-O3)" OFF)
```

Comment:
The PR description doesn't say what the long-term plan is: please add something there.  Is the plan that some outer cmake test suite code is going to figure out whether to turn this on?  Is the plan that this will be on by default soon once some blocking issues are fixed?  Or will this always be some off-by-default option only used manually?

Reply:
This patch is preparation work to defend SVE support because we intend to start precommit testing on SVE hardware in the near future. We will enable the SVE tests in a GitHub Runner hosted by the SVE machine, separate from the existing AArch64 recommit testing.

Reply:
Normally a compiler flag would be auto-enabled after running `CHECK_C_COMPILER_FLAG` so I'm not sure why this is a user-set option?  I would expect this to be auto-set based on the toolchain and platform (we already have `proc_supports_sve`)?

Reply:
OK you mean this will be auto-set long-term, but is a user-set option for preliminary testing?  Please add a TODO comment by this option explaining that this is temporary and will be replaced by auto-setting.